src/templates/static/legal/metatrader/biel.jsx

```diff
@@ -46,7 +46,7 @@ const BIEL = () => (
             <li><strong>{it.L('Opening an MT5 Real Account')}</strong>
                 <p>{it.L('To be able to trade OTC instruments available on the MT5 Trading Platform, you have to first:')}</p>
                 <ol>
-                    <li>{it.L('Open a Binary Options Real Account through our website www.binary.com; and')}</li>
+                    <li>{it.L('Open a Binary Options Real Account through our website [_1]; and', `<a href=${it.url_for('')} target="_blank">www.binary.com</a>`)}</li>
```

Comment:
should we be linking the signup page here?

Reply:
yeah, i think so :)